source/Cosmos.System2/Kernel.cs

```diff
@@ -3,14 +3,15 @@
 using Cosmos.Debug.Kernel;
 using Cosmos.HAL;
 using Cosmos.System.ScanMaps;
+using Cosmos.Core;
```

Comment:
you can not access the core ring in the system ring this will break when the ring master comes out

Reply:
You can remove the "AdvancedObject" class in the core ring

Reply:
Change is reveted